src/bitcoin-cli.cpp

```diff
@@ -88,8 +86,10 @@ static bool AppInitRPC(int argc, char* argv[])
         return false;
     }
     // Check for -testnet or -regtest parameter (BaseParams() calls are only valid after this clause)
-    if (!SelectBaseParamsFromCommandLine()) {
-        fprintf(stderr, "Error: Invalid combination of -regtest and -testnet.\n");
+    try {
+        SelectBaseParams(ChainNameFromCommandLine());
+    } catch(std::exception &e) {
```

Comment:
I don't get it, why such stuff is alway reintroduced or even used at all:
Should be: ``catch (const std::exception&)``